src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -682,32 +755,54 @@ private void fillFlightContexts(Connection connection, List<FlightContext> fligh
   }
 
   /**
-   * Return flight state for a single flight
+   * Return flight state for a single flight. The status of completed flight is limited by
+   * completeFlightAvailable property.
```

Comment:
I don't see the `completeFlightAvailable` property mentioned anywhere else.

Reply:
Does this still need to be removed?